.yamato/project.metafile

```diff
@@ -1,4 +1,4 @@
-validation_editor: 2021.1
+validation_editor: 2020.3
```

Comment:
Is this going to cause us problems when we try to go 1.0.0-pre? Will validation fail on 2020.3 maybe?

Reply:
No, we are fine with UTP matter in fact just to ensure we don't break anything we actually run validation on trunk/2020.3/2021.2 but the tooling just looks for 2020.3 so we ensure we don't break newer validation tools. The upm tools just check if validation has run on at least the min version